webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ChangeHostCVModal.js

```diff
@@ -193,35 +155,7 @@ const ChangeHostCVModal = ({
           onToggle={isExpanded => setCVSelectOpen(isExpanded)}
           placeholderText={cvPlaceholderText()}
         >
-          {contentViewsInEnv?.map(cv => (
-            <SelectOption
-              key={cv.id}
-              value={cv.id}
-            >
-              <Flex
-                direction={{ default: 'row', sm: 'row' }}
-                flexWrap={{ default: 'nowrap' }}
-                alignItems={{ default: 'alignItemsCenter', sm: 'alignItemsCenter' }}
-              >
-                <ContentViewIcon
-                  composite={cv.composite}
-                  size="sm"
-                />
-                <Flex
-                  direction={{ default: 'column', sm: 'column' }}
-                  flexWrap={{ default: 'nowrap' }}
-                  alignItems={{ default: 'alignItemsFlexStart', sm: 'alignItemsFlexStart' }}
-                >
-                  {cv.name}
-                  <ContentViewDescription
-                    cv={cv}
-                    versionNumber={relevantVersionFromCv(cv, selectedEnv)}
-                  />
-                </Flex>
-              </Flex>
-            </SelectOption>
-          ))
-          }
+          {contentViewsInEnv?.map(cv => ContentViewSelectOption(cv, selectedEnvForHost[0]))}
```

Comment:
```suggestion
          {contentViewsInEnv?.map(cv => <ContentViewSelectOption cv={cv} selectedEnvForHost={selectedEnvForHost[0]} />)}
```

Reply:
Updating it this way makes the selected option show up as Id instead of name for some reason. Pushed a few updates other than this change.

Reply:
you'll also need to make the function take an object ;)

Reply:
```js
<ContentViewSelectOption cv={cv} />
```

is the same as
```js
ContentViewSelectOption({ cv })
```

or 
```js
ContentViewSelectOption({ cv: cv })
```